handlers/reg.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+
```

Comment:
Any reason for 'reg' instead of 'registration'?

Reply:
No, it was just shorter to write haha